README.md

```diff
@@ -1,31 +1,188 @@
-Odoo Docker Resources
-=====================
+[![License: MIT](https://img.shields.io/badge/licence-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+[![Build Status](https://travis-ci.org/LasLabs/docker-odoo.svg?branch=master)](https://travis-ci.org/LasLabs/docker-odoo)
+
+Alpine Odoo
+===========
+
+This provides an Odoo instance for Docker using Alpine Linux.
+
+It aims to operate in the exact same way as the official Odoo Docker image,
+although there are also some additional features that are helpful for developers
+intending to use OCA modules.
```

Comment:
I'm pretty sure to is correct here - `developers intending the use OCA modules` sounds weird IMO